web/vite.config.ts

```diff
@@ -16,7 +16,21 @@ const config = (mode: string) => {
 			include: ['**.test.ts']
 		},
 
+		server: {
+			port: parseInt(process.env['WEB_SERVER_PORT'] ?? '') || 3000,
+			proxy: {
+				'/graphql': {
+					target: 'ws://localhost:4000/v1',
+					ws: true
+				},
+				'/hasura': {
+					target: 'http://localhost:4000/console'
+				}
+			}
```

Comment:
Am also proxying these to make it easier for it to work on LAN (maybe there are better ways to do it)